### DISCUSSION TITLE
Too few arguments to function Spatie\Ray\Settings\Settings::__construct(), 0 passed

### DISCUSSION THREAD

**Discussion by smail-aouissa** [spatie/laravel-settings#125] [Q&A]
Hi,

I followed all the installation steps, but container injection does not work with me, I receive : "Too few arguments to function ..., 0 passed" with both cases as parameter or using app helper

<img width="1117" alt="Screen Shot 2022-03-25 at 21 13 36" src="https://user-images.githubusercontent.com/11202352/160194581-0add4832-84de-453c-b825-237f7ebfe313.png">

Laravel 8
PHP 7.4

**Comment by rubenvanassche** [comment]
You're not using the settings class from this package but from the Ray package. Importing the correct settings class should solve the issue.
